sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # Release History
 
-## 12.7.0 (Unreleased)
+## 12.7.0-beta.1 (Unreleased)
 
 ### Features Added
 
+- Updated Azure Storage Service API version to 2020-10-02.
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 - Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
+- Added support for OAuth in copying source in `BlobClient.syncCopyFromURL()`, `BlockBlobClient.syncUploadFromURL()`, `BlockBlobClient.stageBlockFromURL()`, `AppendBlobClient.appendBlockFromURL()` and `PageBlobClient.uploadPagesFromURL()`.
```

Comment:
datalake needs change?

Reply:
Datalake doesn't have copy request. Customer would need to leverage Blob package for copying. So we don't need to add change for datalake.

Reply:
Consult Yang Bo about the DataLake move API on whether the header will be supported there?

Reply:
Tried with this interface, it doesn't support source OAuth Token . source OAuth Token is only supported in sync copy requests, move operation is not a sync copy.